### PR TITLE
Adding "set destination" button to links

### PR DESCRIPTION
**Pull request by gvn**

### CODE REVIEW

**Comment by gvn** [comment]
Related to #1670

File: www_src/components/el/el.jsx [hunk 1]
```diff
@@ -67,10 +67,30 @@ var El = React.createClass({
     if(!prevState.touchactive && this.state.touchactive) {
       this.onUpdate();
     }
+
+    if (this.props.type === 'link') {
```
Comment (gvn) [inline www_src/components/el/el.jsx]: Debating whether this is too business logic-y to have unabstracted within El...

File: www_src/components/el/el.jsx [hunk 2]
```diff
@@ -67,10 +67,30 @@ var El = React.createClass({
...
+  // Button position is based on the rendered DOM, so we're setting it directly post-render
+  positionButton: function () {
+    var elWrapper = this.getDOMNode();
+    var elStyleWrapper = elWrapper.querySelector('.style-wrapper');
+    var elButton = elWrapper.querySelector('.meta-button');
+
+    var boundingBox = elStyleWrapper.getBoundingClientRect();
+    var buttonStyle = this.getInitialState();
+
+    buttonStyle.y += (((boundingBox.bottom - boundingBox.top) / 2) + 40);
```
Comment (thisandagain) [inline www_src/components/el/el.jsx]: ((Going crazy) with the ()) :wink:

**Comment by thisandagain** [comment]
R+ :panda_face: :palm_tree: :soccer: :sheep:

**Pull request merged**